src/controllers/users_controller.js

```diff
@@ -5,23 +5,29 @@ module.exports.setUp = () => {
 };
 
 module.exports.findAllUsers = (req, res) => {
-    res.json(usersModel.findAllUsers());
+    usersModel.findAllUsers((err, users) => {
+        res.status(err ? 500 : 200).json(users);
```

Comment:
If there's an error, why is it sending the array of users down?

Reply:
i'll fix this :)